strawberry/chalice/views.py

```diff
@@ -0,0 +1,127 @@
+import functools
+from http import HTTPStatus
+
+from chalice.app import CaseInsensitiveMapping, Request, Response
+from strawberry.chalice.graphiql import render_graphiql_page
+from strawberry.http import GraphQLHTTPResponse, process_result
+from strawberry.schema import BaseSchema
+from strawberry.types import ExecutionResult
+
+
+class GraphQLView:
+    def __init__(self, schema: BaseSchema, render_graphiql: bool = True):
+        self._schema = schema
+        self.graphiql = render_graphiql
+
+    @staticmethod
+    @functools.lru_cache()
```

Comment:
do you need this if we already have a cache for render_graphiql_page?

Reply:
You're right the other cache covers this - I have now removed it.